### PR TITLE
Maximizes layout width

### PR DESCRIPTION
**Pull request by caseyyee**
image

### CODE REVIEW

File: themes/aframe/source/css/home.styl [hunk 1]
```diff
@@ -17,9 +17,6 @@ ul
 .home
   display flex
   flex-wrap wrap
-  padding 30px
```
Comment (ngokevin) [inline themes/aframe/source/css/home.styl]: I would keep the 30px padding (or slightly smaller), but we can remove the 8% piece.

**Comment by ngokevin** [comment]
r+wc

File: themes/aframe/source/css/home.styl (hunk 1 above)
Comment (caseyyee) [inline themes/aframe/source/css/home.styl]: I was going to make the layout borderless with padding within the content sections itself.   looks jank now, but will fix.

Comment (ngokevin) [inline themes/aframe/source/css/home.styl]: ok

**Pull request merged**